### PR TITLE
GitOps Operator | Update CI version to 4.13

### PR DESCRIPTION
**Pull request by mehabhalodiya**
Update OpenShift CI to run tests on OCP 4.12-4.13

Related: [GITOPS-3162](https://issues.redhat.com/browse/GITOPS-3162)

### CODE REVIEW

File: ci-operator/config/redhat-developer/gitops-operator/redhat-developer-gitops-operator-v1.8__v4.13.yaml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+- context_dir: .
+  from: src
+  to: gitops-operator
+operator:
+  bundles:
+  - as: gitops-operator-bundle
+    context_dir: bundle/
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/redhat-developer/gitops-operator:latest
```
Comment (varshab1210) [inline ci-operator/config/redhat-developer/gitops-operator/redhat-developer-gitops-operator-v1.8__v4.13.yaml:33]:
```suggestion
  - pullspec: quay.io/redhat-developer/gitops-operator:v1.8.0
```

**Comment by mehabhalodiya** [comment]
/pj-rehearse pull-ci-redhat-developer-gitops-operator-master-v4.13-kuttl-parallel

**Pull request merged**